lib/livebook/hubs.ex

```diff
@@ -112,4 +118,84 @@ defmodule Livebook.Hubs do
   defp to_struct(%{id: "local-" <> _} = fields) do
     Provider.load(%Local{}, fields)
   end
+
+  @doc """
+  Connects to the all available and connectable hubs.
+
+  ## Example
+
+      iex> connect_hubs()
+      :ok
+
+  """
+  @spec connect_hubs() :: :ok
+  def connect_hubs do
+    for hub <- fetch_hubs(), Provider.connectable?(hub) do
+      Provider.connect(hub)
+    end
+
+    :ok
+  end
+
+  @doc """
+  Gets a list of connected hubs.
+
+  ## Example
+
+      iex> fetch_connected_hubs()
+      %{"enterprise-12345" => %{pid: #PID<0.178.0>, hub: %Enterprise{}}}
+
+  """
+  @spec fetch_connected_hubs() :: connected_hubs()
+  def fetch_connected_hubs do
+    for hub <- fetch_hubs(), reduce: %{} do
+      acc ->
+        case fetch_connected_hub(hub) do
+          nil -> acc
+          connected_hub -> Map.put_new(acc, hub.id, connected_hub)
+        end
+    end
+  end
+
+  @doc """
+  Gets one connected hub.
+
+  ## Examples
+
+      iex> fetch_connected_hub(%Enterprise{})
+      %{pid: #PID<0.178.0>, hub: %Enterprise{}}
+
+      iex> fetch_connected_hub(%Enterprise{})
+      nil
+
+      iex> fetch_connected_hub(%Fly{})
+      nil
+
+  """
+  @spec fetch_connected_hub(Provider.t()) :: connected_hub() | nil
+  def fetch_connected_hub(hub) do
```

Comment:
Usually the convention is to use `get_` for `term | nil` and `fetch_` for `{:ok, term} | :error` :)